aten/src/ATen/native/cuda/int4mm.cu

```diff
@@ -125,7 +125,11 @@ inline __host__ __device__ uint32_t getAlignmentRoundUp(const void* p) {
   return diff == 0 ? 0 : uint32_t(Align) - diff;
 }
 
+#if defined(USE_ROCM)
+constexpr int32_t kWarpSize = warpSize;
+#else
 constexpr int32_t kWarpSize = 32;
+#endif
```

Comment:
`warpSize` is defined for CUDA as 32, so `if defined` is not required?

Reply:
not sure I am following you? `warpSize` is defined in hip language runtime. We would need `else` for CUDA build

Reply:
@petrex  See below comment.
We use it in host code